enigma-principal/app/src/boot_network/keys_provider_http.rs

```diff
@@ -222,7 +222,7 @@ mod test {
 
     #[test]
     pub fn test_jsonrpc_get_state_keys() {
-        setup_epoch_storage();
+        let path = setup_epoch_storage();
```

Comment:
There is no use of this variable `path`. Is it needed?

Reply:
removed it.